env/static/examples/simple/main.tf

```diff
@@ -0,0 +1,6 @@
+module "env" {
+  source = "../../"
```

Comment:
Since it's for usage outside of this repo, does it makes sense to have full git path here?

Reply:
@docktorrr No because this is for CI tests it has to resolve to the current state on disk so that it's testing the latest commit state on the branch.